CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/concept/ContentConcept.java

```diff
@@ -130,7 +132,7 @@ public String getName() {
         public static final SchemaAttribute COMPANY = new SchemaAttribute.Builder(GraphElementType.TRANSACTION, StringAttributeDescription.ATTRIBUTE_NAME, "Content.Company")
                 .setDescription("The company authoring the content")
                 .build();
-        public static final SchemaAttribute CONTENT = new SchemaAttribute.Builder(GraphElementType.TRANSACTION, StringAttributeDescription.ATTRIBUTE_NAME, "Content")
+        public static final SchemaAttribute CONTENT = new SchemaAttribute.Builder(GraphElementType.TRANSACTION, StringAttributeDescription.ATTRIBUTE_NAME, CONTENT_NAME)
```

Comment:
Although the string is the same, this use of CONTENT is different to above, i'd keep these separate to highlight that point.

Reply:
Reverted. No point having the constant for just the name